_episodes/03-terms.md

```diff
@@ -17,65 +17,6 @@ keypoints:
 - "Pedagogical content knowledge: connects general understanding of teaching to domain-specific content."
 ---
 
-This section outlines the structure of this two-day course and provides some
-essential terms for discussion.
-
-## Practice and Feedback
```

Comment:
Could you move the 'Practice and Feedback' part to the beginning of 05-performance?